molecule/default/verify.yml

```diff
@@ -20,6 +20,16 @@
       - item.name in aiida_pps.stdout
     with_items: "{{ aiida_pseudopotentials }}"
 
+  - name: print aiida-core version
+    shell: "{{ aiida_venv }}/bin/pip show aiida-core"
+    changed_when: false
```

Comment:
I think you don't even need this in `verify.yml` (of course fine to keep)

Reply:
I thought just in case a plugin tries to install a different version

Reply:
Sorry, I meant the `changed_when` is not needed (it's fine to change things in `verify.yml`)